constants/parameters.php

```diff
@@ -224,7 +224,18 @@
 const ISSUE_ALIASES = ['issue', 'number'];
 const PAGE_ALIASES = ['page', 'pages', 'pp', 'p', 'at'];
 
-
+// Just leave this code here for now
```

Comment:
This comment should give a rationale for leaving the code, and a note specifying when the code should be removed (or it'll be there forever...)